lib/src/config.rs

```diff
@@ -314,6 +325,21 @@ impl ConfigLayer {
     ) -> Result<Option<&ConfigValue>, &ConfigValue> {
         look_up_item(&self.data.cache, name.into_name_path().borrow())
     }
+
+    /// Inserts new `value` at the `name` path.
+    pub fn insert_value(
```

Comment:
Would it make sense to call it `set_value()`? The goal is to make it not sound like it's going to insert it in any particular position. Because there's no ordering implied within a layer, right? I'm guessing you used "insert" in the `HashMap::insert` sense (I'm not sure why that isn't called  "set").

Reply:
Yes, I just copied `insert` because the underlying function is called as such. `set` also works.